static_src/test/functional/overview_quicklook.spec.js

```diff
@@ -43,7 +43,9 @@ describe('Overview page', function () {
     });
 
     it('is expanded', function () {
-      expect(quicklookElement.isExpanded()).toBe(true);
+      browser.waitUntil(function() {
+        return quicklookElement.isExpanded();
+      }, 2000, 'quicklook does not expand');
```

Comment:
We should add an `expand()` method to the `quicklook.element` that calls `click` and then does the wait. This kind of stuff should be encapsulated in the page objects since the wait will be needed anytime you want to trigger the click in a test.